src/ResourceManager/DataLakeStore/Commands.DataLakeStore/Commands.DataLakeStore.csproj

```diff
@@ -116,7 +126,15 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
     <None Include="MSSharedLibKey.snk" />
-    <None Include="packages.config" />
+    <Content Include="NLog.config">
```

Comment:
@rahuldutta90 what are `NLog.config` and `NLog.xsd` being used for?

Reply:
This is not mandatory. It is used to configure NLog. I had a dependency on NLog.config which was not needed. For future release I have removed this dependency. Here I have manually removed these files, updated the csproj and packages.config